src/JabbR-Core/appsettings.json

```diff
@@ -11,7 +20,7 @@
     }
   },
   "Data": {
-    "ConnectionStrings": { 
+    "ConnectionStrings": {
```

Comment:
Lets keep a named connection out of here, since it will be encrypted/environment variable only. This whole section Data could be removed.